File: app/src/main/java/com/legue/axel/lolsummonertool/database/model/Rune.java
package com.legue.axel.lolsummonertool.database.model;

import com.google.gson.annotations.SerializedName;

public class Rune {

    @SerializedName("isrune")
    public boolean isRune;
    public int tier;
    public String type;

    public Rune() {
    }

    public Rune(boolean isRune, int tier, String type) {
        this.isRune = isRune;
        this.tier = tier;
        this.type = type;
    }

}

File: app/src/main/java/com/legue/axel/lolsummonertool/database/model/Rune.kt
package com.legue.axel.lolsummonertool.database.model

import com.google.gson.annotations.SerializedName

data class Rune(
        @SerializedName("isrune")
        var isRune: Boolean = false,
        var tier: Int = 0,
        var type: String? = null

)